src/middlewared/middlewared/plugins/pool_/import_pool.py

```diff
@@ -338,6 +338,22 @@ def import_on_boot(self, job):
             # event logic
             return
 
+        # Attach NVMe/RoCE - wait up to 10 seconds
+        self.logger.info('Start bring up of NVMe/RoCE')
```

Comment:
We don't need to attach JBOFs on non-ix sold hardware. We can put a conditional here `truenas.is_ix_hardware`